Frameworks/include/CGPathInternal.h

```diff
@@ -56,5 +57,6 @@ struct CGPathElementInternal : CGPathElement {
 typedef struct CGPathElementInternal CGPathElementInternal;
 
 HRESULT _CGPathGetGeometry(CGPathRef path, ID2D1Geometry** pGeometry);
+HRESULT _CGPathApplyInternal(Microsoft::WRL::ComPtr<ID2D1PathGeometry> pathGeometry, void* info, CGPathApplierFunction function);
 
```

Comment:
_Please_ only pass raw com pointers at TU bounaries. It'll let us drop the WRL dependency from CGPathInternal, and avoid some unnecessary addref/release/copy construction on the ComPtr.